OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs

```diff
@@ -39,7 +40,7 @@ public class TakeCoverInfo : TurretedInfo
 		public override object Create(ActorInitializer init) { return new TakeCover(init, this); }
 	}
 
-	public class TakeCover : Turreted, INotifyDamage, IDamageModifier, ISpeedModifier, ISync, IRenderInfantrySequenceModifier
+	public class TakeCover : Turreted, INotifyDamage, IDamageModifier, ISpeedModifier, ISync, IRenderInfantrySequenceModifier, IObservesVariables, IDisabledTrait
```

Comment:
Similarly you don't need `, IObservesVariables, IDisabledTrait`.

Reply:
leftovers when I originally made it